lib/services/sessiontracker.go

```diff
@@ -44,3 +50,118 @@ type SessionTrackerService interface {
 	// UpdatePresence updates the presence status of a user in a session.
 	UpdatePresence(ctx context.Context, sessionID, user string) error
 }
+
+// UpdateSessionTrackerState is a helper function to add a session tracker participant.
```

Comment:
```suggestion
// AddSessionTrackerParticipant adds a session tracker participant.
```

Reply:
As a side note, a bunch of these gods aren't adding much value, so feel free to delete if you prefer.